web-app/packages/admin-lib/src/modules/admin/views/ProjectView.vue

```diff
@@ -0,0 +1,222 @@
+            {{ project?.name }}
+          </h2>
+          <dl
+            class="project-view-detail-list paragraph-p5 flex flex-column gap-3"
+          >
+            <div>
+              <dt class="paragraph-p6 opacity-80">Created</dt>
+              <dd class="font-semibold" data-cy="project-owner">
+                {{ $filters.date(project?.created) }}
+              </dd>
+            </div>
+            <div>
+              <dt class="paragraph-p6 opacity-80">Updated</dt>
+              <dd class="font-semibold" data-cy="project-created">
+                {{ $filters.date(project?.updated) }}
+              </dd>
+            </div>
+            <div>
+              <dt class="paragraph-p6 opacity-80">Disk usage</dt>
+              <dd class="font-semibold" data-cy="project-created">
+                {{ $filters.filesize(project?.disk_usage, 'MB') }}
+              </dd>
+            </div>
+          </dl>
+        </div>
+      </app-section>
+    </app-container>
+
+    <app-container>
+      <app-section>
+        <template #title>Advanced</template>
+        <div class="flex flex-column row-gap-3 paragraph-p5 px-4 pb-4">
+          <div
+            :class="[
+              'flex flex-column align-items-start',
+              'row-gap-2',
+              'md:align-items-center md:flex-row'
+            ]"
+          >
+            <div class="flex-grow-1">
+              <template v-if="project?.access?.public">
+                <p class="title-t3">This is public project</p>
+                <span class="paragraph-p6 opacity-80"
+                  >Hide this project from everyone.</span
+                >
+              </template>
+              <template v-else>
+                <p class="title-t3">This is private project</p>
+                <span class="paragraph-p6 opacity-80"
+                  >Make this project visible to anyone.</span
+                >
+              </template>
+            </div>
+            <div class="flex-shrink-0">
+              <PButton
+                @click="confirmPublicPrivate()"
+                severity="secondary"
+                data-cy="settings-public-btn"
+                :label="
+                  project?.access?.public ? 'Make private' : 'Make public'
+                "
+              />
+            </div>
+          </div>
+
+          <div
+            :class="[
+              'flex flex-column align-items-start',
+              'row-gap-2',
+              'md:align-items-center md:flex-row'
+            ]"
+          >
+            <div class="flex-grow-1">
+              <p class="title-t3">Delete project</p>
+              <span class="paragraph-p6 opacity-80"
+                >Deleting this project will remove it and all its data. This
+                action cannot be undone.</span
+              >
+            </div>
+            <div class="flex-shrink-0">
+              <PButton
+                @click="confirmDelete"
```

Comment:
Do we have both, soft and hard delete options?

Reply:
I intrdoduced here hard delete only option. I am not sure If we want to have both in admin.